scripts/download_nghttp2.sh

```diff
@@ -4,5 +4,4 @@
 set -ex
 
 # Clone the repository to the specified directory.
-git clone --branch v1.33.0 https://github.com/nghttp2/nghttp2 $1
-
+git clone --depth=1 --branch v1.35.x https://github.com/nghttp2/nghttp2 $1
```

Comment:
`v1.35.x`? I think you should pick either .0 or .1 as per https://github.com/nghttp2/nghttp2/releases

Reply:
Aye!